doc/gallery/geospatial/geopoints.ipynb

```diff
@@ -56,7 +67,8 @@
     "    ylabel='Latitude',\n",
     "    alpha=0.5,\n",
     "    color='mag',\n",
-    "    cmap='viridis_r'\n",
+    "    cmap='viridis_r',\n",
+    "    features=[\"coastline\"]\n",
```

Comment:
I noticed that adding the coastline has made the matplotlib plot much smaller. Maybe you can add a `width` parameter there?